dir.targets

```diff
@@ -85,4 +74,10 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="ReportGroups"
+          BeforeTargets="PrepareForBuild"
```

Comment:
Why PrepareForBuild? I'm not sure that if that is right or wrong just asking why you picked that one?